govuk-prototype-kit.config.json

```diff
@@ -7,7 +7,7 @@
   },
   "pluginDependencies": [{
     "packageName": "govuk-frontend",
-    "minVersion": "4.4.0"
+    "minVersion": "5.0.0"
```

Comment:
Thanks for bumping this one too. Does it definitely work?

i.e. For prototypes on GOV.UK Frontend v4.7.0, does updating this plugin also apply the v5.0.0 update?

Still trying to figure out which bits of `minVersion` and `maxVersion` were implemented

Reply:
So from doing a quick local test of downgrading govuk-frontend in a prototype to 4.3.0, it doesn't _seem_ to throw any warning at all.

I think that if this plugin was updated before updating `govuk-frontend` to `5.0.0`+, nothing would immediately crash, you'd just get an error if you tried to use the Task list template before updating?